cli/release.go

```diff
@@ -149,3 +156,31 @@ func runReleaseAddDocker(args *docopt.Args, client *controller.Client) error {
 
 	return nil
 }
+
+func canonicalizeDockerURI(rawurl string) (string, error) {
+	log.Println("Validating Docker image URI.")
+	u, err := url.Parse(rawurl)
+	if err != nil {
+		return "", err
+	}
+	if u.Host != "registry.hub.docker.com" {
+		return "", fmt.Errorf(`Invalid registry "%s"`, u.Host)
+	}
+	q := u.Query()
+	ref, err := registry.NewRef(rawurl)
+	if err != nil {
+		return "", err
+	}
+	image, err := registry.NewDockerSession(ref).GetImage()
+	if err != nil {
+		return "", err
+	}
+	q.Set("id", image.ID)
+	if tag := q.Get("tag"); tag != "" {
+		q.Del("tag")
+		log.Printf("Resolved tag %s => %s", tag, u.String())
+	}
+	u.RawQuery = q.Encode()
+	log.Println("Docker image URI is valid!")
```

Comment:
I don't think this is a necessary log message, successfully creating the release after telling the user the URI is being validated seems enough:

```bash
Validating Docker image URI
Created release 427537e78be4417fae2e24d11bc993eb.
```